mapper/dm_agent/src/batcher.rs

```diff
@@ -222,28 +234,21 @@ mod tests {
     use mockall::Sequence;
     use tokio::time::sleep;
 
-    use super::*;
-
     #[test]
     fn test_message_batch_processor() -> anyhow::Result<()> {
-        let topic = Topic::new("collectd/localhost/temperature/value").unwrap();
-        let collectd_message = Message::new(&topic, "123456789:32.5");
+        let collectd_message = CollectdMessage::new("temperature", "value", 32.5);
```

Comment:
This might be tested elsewhere, but we are reducing the test coverage here.

I would expect:
```
let topic = Topic::new("collectd/localhost/temperature/value").unwrap();
let message = Message::new(&topic, "123456789:32.5");
let collectd_message = parse_from(&message);
```

Reply:
We already have a test that validates exactly that in the `collectd_message_parsing` test in `collectd.rs`.